tutorial/01_basic/README.rst

```diff
@@ -3,19 +3,112 @@
 Basic usage
 ===========
 
-Reading and plotting 3D data using ``examples`` module and external files.  (10 min for talk, 10 min for exercise)
+Reading and plotting 3D data using ``examples`` module and external files.  (10
+min for talk, 10 min for exercise)
 
 .. tip::
 
-    This section of the tutorial was adopted from <insert link>
-    chapter of the PyVista documentation.
+    This section of the tutorial was adopted from the `Basic API Usage
+    <https://docs.pyvista.org/user-guide/simple.html>`_ chapter of the PyVista
+    documentation.
 
 
-.. insert section content here
+Using Exising Data
+~~~~~~~~~~~~~~~~~~
+There are two main ways of getting data into PyVista: creating it yourself from
+scratch or loading the dataset from any one of the `compatible file formats
+<https://docs.pyvista.org/api/readers/index.html>`_. Since we're starting out
+here, let's just load a file.
```

Comment:
```suggestion
Using Existing Data
~~~~~~~~~~~~~~~~~~~
There are two main ways of getting data into PyVista: creating it yourself from
scratch or loading the dataset from any one of the `compatible file formats
<https://docs.pyvista.org/api/readers/index.html>`_. Since we're starting out
here, let's just load a file.
```